.github/workflows/cd.yaml

```
@@ -17,6 +17,7 @@ on:
 concurrency:
   group: publish-skills
   cancel-in-progress: false
+  queue: max
```

actionlint (1.7.12)

[error] unexpected key "queue" for "concurrency" section. expected one of "cancel-in-progress", "group"

(syntax-check)